HexRaysPyTools/core/helper.py

```diff
@@ -20,7 +21,11 @@ def is_imported_ea(ea):
 
 
 def is_code_ea(ea):
-    if idaapi.cvar.inf.procname == "ARM":
```

Comment:
there is same change at line 46 to be applied

Reply:
Good catch, thanks!

I actually hit that case a few hours after those commits but only fixed it in-place (doh!)

Thanks for this tool btw!

Reply:
Would you like to change it, or shall I? Either is fine, but I am AFK at the moment

Reply:
I'm not any related to authors, was looking if anyone have already updated to ida 9 =)